.mergify.yml

```diff
@@ -1,10 +0,0 @@
-pull_request_rules:
```

Comment:
@whatnick current mergify is specific to travis-ci rule. Need to see an example for GHA